Jfb 1

chingiz-mazhitov opened the pull request.

chingiz-mazhitov committed:
add feature to generate 10 tickets

chingiz-mazhitov committed:
add feature to generate 10 tickets in the main method

chingiz-mazhitov committed:
add getTicketById method

Alexmansar commented on src/sample/complete/TicketService.java:
```diff
@@ -1,29 +1,44 @@
-		System.out.println("-".repeat(80));
+		// generate 10 tickets
+		IntStream.rangeClosed(1, 10)
+				.forEach(i -> {
```
what is i? an id, a ticket or something else? please give a more understandable name

Alexmansar commented on src/sample/complete/TicketService.java:
```diff
@@ -1,29 +1,44 @@
 
 	}
+
+	public static Ticket getTicketById(int id) {
```
could be private

Alexmansar commented on src/sample/complete/TicketService.java:
```diff
@@ -1,29 +1,44 @@
+//			System.out.println(value);
+//		}
 
+		var ticket = getTicketById(1);
```
you  can use var, but try noe wothout it please

Alexmansar reviewed (changes requested):
@chingiz-mazhitov Hi, please fix small remarks))

chingiz-mazhitov committed:
give more descriptive name to a parameter
in a for each operation, change access modifier to private for getTicketById method

Alexmansar commented on src/sample/complete/TicketService.java:
```diff
@@ -1,29 +1,44 @@
-		// limited
-		Ticket ticket1 = new Ticket("Couchella", 123, unixTimestamp);
-		System.out.println(ticket1);
+//		for (var value: tickets.values()) {
+//			System.out.println(value);
+//		}
```
please, or remove or uncommented it)

chingiz-mazhitov committed:
remove commented lines

Alexmansar reviewed (approved):
@chingiz-mazhitov please, merge it to develop)
